src/python/diff3_analysis.py

```diff
@@ -0,0 +1,259 @@
+"""
+Recreates a merge and outputs the diff files for two algorithms for comparison on a given conflict.
```

Comment:
Thanks, just added a few points on this in the README:
1. It uses diff3 to show the base, conflicting branches, and the programmer merge which is useful to see the context around the conflict.
2. It automates everything making the workflow faster. You just give it a conflict, and it quickly outputs the differences between a pair of merge algorithms in the .txt file format.

Reply:
This has not answered the question.  Why can't this script either call `replay_merge.py` internally and then do the diff?  That would make this script much shorter and simpler.

Reply:
You should be able to do that, it allows you to delete all the setup code since replay_merge does everything. I would suggest maybe just remove this script and adding a flag `-create_diffs` to `replay_merge.py` that computes the diff as you do it here when this flag is used (by default set it to false though).